src/fairseq2/models/wav2vec2/asr/archs.py

```diff
@@ -14,6 +14,25 @@
 def _base_10h() -> Wav2Vec2AsrConfig:
     return Wav2Vec2AsrConfig()
 
+##################################################################
+# Register the new wav2vec2_asr_arch here which returns a Wav2Vec2AsrConfig: 
+# Here you will change adapt the wav2vec2 config for ASR finetuning.
+@wav2vec2_asr_arch("mms_base_300m_asr")
+def _mms_base_300m_eng_accent() -> Wav2Vec2AsrConfig:
+    config = _base_10h()
```

Comment:
Again, can you initialize this with `_large_lv60k_10h()` and change whatever necessary?